### PR TITLE
feat(op-chain-ops): add Wright fork config into genesis file generation code

### PR DESCRIPTION
**Pull request by welkin22**
### Description

This PR is made in conjunction with https://github.com/bnb-chain/op-geth/pull/130. A new Wright hard fork has been introduced in op-geth. Whenever there is a new hard fork, we need to make corresponding changes in the genesis file generation code to support the activation of the hard fork through the configuration file.

### Rationale

This modification allows us to configure when to activate the hard fork through the configuration file, which is beneficial for both Devnet and Qanet.

### Example

none

### Changes

Notable changes:
* add Wright fork config into genesis file generation code

### CODE REVIEW

**Comment by welkin22** [comment]
Please do not merge this PR yet. We will merge it after the official release of op-geth.

**Review by owen-reorg — APPROVED** [review]

**Review by redhdx — APPROVED** [review]

**Pull request merged**
